coderd/idpsync/organization.go

```diff
@@ -141,6 +189,7 @@ func (s AGPLIDPSync) SyncOrganizations(ctx context.Context, tx database.Store, u
 	}
 
 	if len(notExists) > 0 {
+		notExists = slice.Unique(notExists) // Remove dupes
```

Comment:
```suggestion
		notExists = slice.Unique(notExists) // Remove duplicates
```

comments deserve full words!